### PR TITLE
Fixing code style for pinot-plugins

### PR DESCRIPTION
**Pull request by xiangfu0**
## Description
Fixing code style for all pinot-plugins per #7309
- it requires changing packages for minion tasks

## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

File: pinot-plugins/pinot-file-system/pinot-adls/src/test/java/org/apache/pinot/plugin/filesystem/test/ADLSGen2PinotFSTest.java [hunk 1]
```diff
@@ -47,7 +47,13 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.any;
```
Comment (Jackie-Jiang) [inline pinot-plugins/pinot-file-system/pinot-adls/src/test/java/org/apache/pinot/plugin/filesystem/test/ADLSGen2PinotFSTest.java:50]: Reformat

File: pinot-plugins/pinot-stream-ingestion/pinot-kafka-0.9/src/test/java/org/apache/pinot/plugin/stream/kafka09/KafkaPartitionLevelConsumerTest.java [hunk 2]
```diff
@@ -56,45 +56,45 @@
...
+    private String _topicName;
+    private BrokerEndPoint[] _brokerArray;
 
     public MockKafkaSimpleConsumerFactory(String[] hosts, int[] ports, long[] partitionStartOffsets,
         long[] partitionEndOffsets, int[] partitionLeaderIndices, String topicName) {
       Preconditions.checkArgument(hosts.length == ports.length);
-      this.hosts = hosts;
-      this.ports = ports;
-      brokerCount = hosts.length;
+      this._hosts = hosts;
```
Comment (Jackie-Jiang) [inline pinot-plugins/pinot-stream-ingestion/pinot-kafka-0.9/src/test/java/org/apache/pinot/plugin/stream/kafka09/KafkaPartitionLevelConsumerTest.java:70]: Remove `this`

File: pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaStreamLevelStreamConfig.java [hunk 3]
```diff
@@ -129,7 +129,7 @@ public String getBootstrapServers() {
   }
 
   static {
```
Comment (Jackie-Jiang) [inline pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaStreamLevelStreamConfig.java:131]: Move this to the front?

File: pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/server/KafkaDataProducer.java [hunk 4]
```diff
@@ -50,7 +50,7 @@ public void init(Properties props) {
     props.remove("request.required.acks");
     props.remove("serializer.class");
     try {
-      this.producer = new KafkaProducer<>(props);
+      this._producer = new KafkaProducer<>(props);
```
Comment (Jackie-Jiang) [inline pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/server/KafkaDataProducer.java:53]: Remove `this.`

File: pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/test/java/org/apache/pinot/plugin/stream/kinesis/KinesisConsumerTest.java [hunk 5]
```diff
@@ -57,10 +57,10 @@
   public static final String PLACEHOLDER = "DUMMY";
   public static final int MAX_RECORDS_TO_FETCH = 20;
 
-  private static KinesisConnectionHandler kinesisConnectionHandler;
-  private static StreamConsumerFactory streamConsumerFactory;
-  private static KinesisClient kinesisClient;
-  private List<Record> recordList;
+  private static KinesisConnectionHandler _kinesisConnectionHandler;
```
Comment (Jackie-Jiang) [inline pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/test/java/org/apache/pinot/plugin/stream/kinesis/KinesisConsumerTest.java:60]: These should not be static

File: pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/test/java/org/apache/pinot/plugin/stream/kinesis/KinesisStreamMetadataProviderTest.java [hunk 6]
```diff
@@ -50,10 +50,10 @@
   public static final String CLIENT_ID = "dummy";
   public static final int TIMEOUT = 1000;
 
-  private static KinesisConnectionHandler kinesisConnectionHandler;
-  private KinesisStreamMetadataProvider kinesisStreamMetadataProvider;
-  private static StreamConsumerFactory streamConsumerFactory;
-  private static PartitionGroupConsumer partitionGroupConsumer;
+  private static KinesisConnectionHandler _kinesisConnectionHandler;
```
Comment (Jackie-Jiang) [inline pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/test/java/org/apache/pinot/plugin/stream/kinesis/KinesisStreamMetadataProviderTest.java:53]: Remove `static`

**Review by Jackie-Jiang — APPROVED** [review]
LGTM otherwise

File: pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroUtils.java [hunk 7]
```diff
@@ -332,6 +332,7 @@ private static void extractSchemaWithComplexTypeHandling(org.apache.avro.Schema
       default:
         DataType dataType = AvroSchemaUtil.valueOf(fieldType);
         addFieldToPinotSchema(pinotSchema, dataType, path, true, fieldTypeMap, timeUnit);
+        break;
```
Comment (jackjlli) [inline pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroUtils.java:335]: Is this break statement needed?

File: pinot-plugins/pinot-input-format/pinot-avro/src/main/java/org/apache/pinot/plugin/inputformat/avro/KafkaAvroMessageDecoder.java [hunk 8]
```diff
@@ -189,21 +189,21 @@ private String hex(byte[] bytes) {
 
   private static class SchemaFetcher implements Callable<Boolean> {
     private org.apache.avro.Schema _schema;
-    private URL url;
+    private URL _url;
     private boolean _isSuccessful = false;
 
     SchemaFetcher(URL url) {
-      this.url = url;
+      this._url = url;
```
Comment (jackjlli) [inline pinot-plugins/pinot-input-format/pinot-avro/src/main/java/org/apache/pinot/plugin/inputformat/avro/KafkaAvroMessageDecoder.java:196]: remove `this.`?

File: pinot-plugins/pinot-stream-ingestion/pinot-kafka-0.9/src/test/java/org/apache/pinot/plugin/stream/kafka09/KafkaPartitionLevelConsumerTest.java [hunk 9]
```diff
@@ -56,45 +56,45 @@
...
+      this._ports = ports;
```
Comment (jackjlli) [inline pinot-plugins/pinot-stream-ingestion/pinot-kafka-0.9/src/test/java/org/apache/pinot/plugin/stream/kafka09/KafkaPartitionLevelConsumerTest.java:71]: Same here.

**Review by jackjlli — COMMENTED** [review]
Minor but LGTM.

File: pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroUtils.java (hunk 7 above)
Comment (xiangfu0) [reply pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroUtils.java:329]: this is required from the checkstyle rules

**Pull request merged**